### DISCUSSION TITLE
UI Chat Conservation simple

### DISCUSSION THREAD

**Discussion by vnk8071** [imartinez/privateGPT#238] [Ideas]
This user interface can:
- Make the conservation of chat
- Edit prompt for instruction
- Edit config model, number context
- Clear history chat
<img width=100% src="https://user-images.githubusercontent.com/78080480/238135659-ea0d07d9-8372-4fc3-80ed-20d804e77972.PNG">

**Comment by antsh3k** [comment]
Hey was there a repository link to this UI?

**Reply by vnk8071** [reply]
I have created PR and await approval [PR#91](https://github.com/imartinez/privateGPT/pull/91). You can try my UI have committed.

**Comment by vnk8071** [comment]
New update with my repo for quick trial [https://github.com/vnk8071/E2E-AI-Chatbot](https://github.com/vnk8071/E2E-AI-Chatbot)
